.gitignore

```diff
@@ -14,3 +14,4 @@ third_party/hdrvdp-2.2.2.zip.tmp
 # Output plots
 tools/benchmark/metrics/plots
 tools/benchmark/metrics/results.csv
+tools/conformance/__pycache__
```

Comment:
This is relatively minor but would it be possible to maintain the final `\n`? It makes `cat` better-behaved, for example.

Reply:
sure, makes sense, I didn't intend to no to have the newline..